pkg/metrics/grafana/tidb.json

```diff
@@ -13902,6 +13902,112 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "type": "graph",
+          "title": "Infoschema Cache v2",
+          "gridPos": {
+            "x": 12,
+            "y": 31,
+            "w": 12,
+            "h": 7
+          },
+          "id": 23763572012,
+          "targets": [
+            {
+              "expr": "sum(rate(tidb_domain_infoschema_v2_cache{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (type)",
```

Comment:
wound be better to calculate a hit/miss/evict percent

Reply:
https://github.com/pingcap/tidb/pull/54268/commits/11b48a420c8ddc5dc91024149c95e9981a5c4e26

Reply:
![image](https://github.com/pingcap/tidb/assets/24713065/20acbb1c-f0e0-4b1b-a79b-a6922a010a6f)